apps/extension/src/components/modalPop/ModalPop.tsx

```
@@ -0,0 +1,92 @@
+import {
+  CategoryDropDown,
+  CommonBtn,
+  InfoBox,
+  TextArea,
+  TextFieldPopup,
+  TimePicker,
+  ToggleButton,
+} from '@pinback/design-system/ui';
+import ModalHeader from './ModalHeader';
+interface ModalPopProps {
+  text?: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)